src/halfpipe/tui/general_widgets/select_or_input_path.py

```diff
@@ -156,7 +156,7 @@ async def on_key(self, event: events.Key) -> None:
         """This allows to typing in the prompt when the focus is on the overlay."""
         if any(mod in event.key for mod in ["ctrl", "alt", "shift"]):
             return
-
+        print("//////////////////////////////////////////", event.key)
```

Comment:
Can you make sure that we don't have any print statements remaining?

Reply:
sorry, accidentally  pushed also with undeleting prints, should be ok now